db/fixtures/miq_shortcuts.yml

```diff
@@ -35,20 +35,10 @@
   :rbac_feature_name: catalog
   :startup: true
 - :name: vms_myvms
-  :description: Services / Workloads / VMs & Instances
```

Comment:
@GregP these should not be deleted, intent of having these per accordion was so that user can set a specific accordion to be active upon login.